forte/data/caster.py

```diff
@@ -78,3 +78,43 @@ def input_pack_type():
     @staticmethod
     def output_pack_type():
         return MultiPack
+
+
+class DataPackBoxer(Caster[MultiPack, DataPack]):
+    """
+    This class creates a DataPack from a MultiPack, this DataPack is the only
```

Comment:
Check the grammar too.

`this DataPack is the only contain` is a bit confusing